hugo-runtime/src/main/java/hugo/weaving/internal/Hugo.java

```diff
@@ -72,6 +80,10 @@ private static void enterMethod(JoinPoint joinPoint) {
   }
 
   private static void exitMethod(JoinPoint joinPoint, Object result, long lengthMillis) {
+    if (android.os.Build.VERSION.SDK_INT >= 18) {
```

Comment:
Any reason you didn't import the `Build` class? Also why not use `Build.VERSION_CODES.JELLY_BEAN_MR2` here instead of 18?